_integration-test/run.sh

```diff
@@ -130,6 +130,42 @@ cat debug.log
 echo '------------------------------------------'
 echo "${_endgroup}"
 
+echo "${_group}Test symbolicator works ..."
+SENTRY_ORG="${SENTRY_ORG:-sentry}"
```

Comment:
Under what circumstance are `SENTRY_ORG`, `SENTRY_PROJECT`, or `SENTRY_TEAM` not declared for integration tests?

Reply:
They aren't by default, but I am mimicking the line above where `SENTRY_TEST_HOST` gets set.